service/src/main/java/uk/gov/hmcts/reform/pebble/EmergencyProtectionOrderDirectionMappingFilter.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.pebble;
```

Comment:
Please move all mapping filters under `uk.gov.hmcts.reform.fpl.pebble` (see FPL inside).

A `uk.gov.hmcts.reform.pebble` is supported to contain only classes that are not FPL specific and could be potentially exported as separate module.